### DISCUSSION TITLE
Request: Show Hierarchy of Parent/Child Location

### DISCUSSION THREAD

**Discussion by MenooMeow** [sysadminsmedia/homebox#381] [Ideas]
I created a bunch of locations that represent a room and storage box for items to be located in.  But, I cannot specify the whole hierarchy of locations at the same time for an item.  For example:
**O-Ring Puller Tool**  --> Located in -->  **Garage > Rack 2 > Box 12**
In my case, I store things in individual boxes or containers, which are located in a specific bin/rack, which are inside a specific room. 
Right now I can only specify that the item is located in "Garage".

**Comment by tankerkiller125** [comment]
Locations support Parents, as do items. Which can create trees in the tree view.

**Reply by MenooMeow** [reply]
Thanks for responding.  I figured out what was wrong...  I think it's a bug.  When I create a node in the tree, it doesn't show up until I manually reload the page in the browser.  After that, the nested node appears in the Location selection box.

**Comment by Viper00o** [comment]
How do you organize this tree?
![image](https://github.com/user-attachments/assets/79834ed7-83d4-4662-bb53-196688eea04a)

![image](https://github.com/user-attachments/assets/9a25ecd3-d312-4f3c-8964-659260c88d01)
There seems to be a conflict for Items within Items. In the  above graphic you can see there are fields for both Parent Location & Parent Item. When you specify the Parent Item the Parent Location should be where the parent is located. However there is the possibility of specifying the wrong location for the parent in this form. The parent location is already in the database. The child should inherit the parents location.
